.gitignore

```diff
@@ -0,0 +1,2 @@
+.history
+text.md
```

Comment:
Please create a `test.md` file and mention the file name `test.md` here in .gitignore file 👍🏻 
Please correct the spelling, It's the `test.md` file not `text.md` 👍🏻

Reply:
Fixed it